docs/tutorials/local_iteration_series/local-Iteration-testing-multiplayer-games-locally.md

```diff
@@ -0,0 +1,62 @@
+---
+id: local_iteration_testing_locally
+title: Local Iteration - Testing multiplayer games locally
+description: Guide covering the available workflows for testing multiplayer games locally.
+---
+
+Testing a multiplayer game presents unique challenges to developers. 
+Currently Unity does not have a first-party streamlined solution to the problem of local iteration while developing a multiplayer game. Developers have to either repeatedly create player builds, or they have to use a symbolic link solution which comes with a set of drawbacks and is not officially supported by Unity.
+
+In lieu of a first-party solution we document the practices that can be used today, along with call-outs for specific risks and things to watch out for.
+
+Our goal here is to document the approaches for managing multiplayer development/testing on a local machine that should be "safe enough", meaning that the project integrity is maintained regardless of the workflow and work progress is not lost.
+
+## Player Builds
+
+First we need to build an executable. The default way of doing that is via  `File->Build Settings` in the menu bar, and then pressing `Build` button.
+
+Once the build has completed you can launch several instances of the built executable in order to both host and join a game.
+
+It is also possible to run the builds along with an editor that produced said build, which could be useful during iterations.
+
+> Mac users: to run multiple instances of the same app, you need to use the command line.
+> Run `open -n YourAppName.app`
+
+This approach, though functional, is somewhat inconvenient and slow - thus we list another option - ParrelSync.
+
+## ParrelSync
+![parrelsync-bossroom-demo](/static/img/parrelsync-bossroom-demo.gif)
+
+[ParrelSync](https://github.com/VeriorPies/ParrelSync) is an open-source Unity editor extension that allows users to **test multiplayer gameplay without building the project** by having another Unity editor window opened and mirror the changes from the original project. 
+
+ParrelSync works by making a copy of the original project folder and creating symbolic links to the `Asset` and `Project Settings` folders back from the original project.
+
+We use ParrelSync for local iteration in [BossRoom sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/).
+
+### Installation
+1.  Backup your project folder or use a version control system such as [Git](https://git-scm.com/), [SVN](https://subversion.apache.org/), [Plastic](https://www.plasticscm.com/) or any other.
```

Comment:
That's needed?
Are those steps part of ParrelSync's install steps? I'm wondering if we should take ownership of these steps, I'd personally just point to their install instructions? @Briancoughlin what do you think?